pkg/handlers/config.go

```diff
@@ -591,7 +591,17 @@ func updateAppConfig(updateApp *apptypes.App, sequence int64, configGroups []kot
 		return updateAppConfigResponse, err
 	}
 
-	if !skipPreflights {
+	hasStrictPreflights, err := store.GetStore().HasStrictPreflights(updateApp.ID, sequence)
+	if err != nil {
+		updateAppConfigResponse.Error = errors.Cause(err).Error()
```

Comment:
let's just return a hardcoded message here? we don't know what the stacktrace will have, and it's not a best practice to return anything from the stacktrace to the user.
```suggestion
		updateAppConfigResponse.Error = "failed to check if version has strict preflights"
```